CommunityToolkit.Mvvm/Messaging/StrongReferenceMessenger.cs

```diff
@@ -24,43 +24,50 @@ public sealed class StrongReferenceMessenger : IMessenger
-    //                   ___(Type2.TToken)____/                                   /          /
-    //                  /________________(Type2.TMessage)________________________/          /
-    //                 /       ____________________________________________________________/
+    //                   |                 \________________[*]IDictionary2<Recipient, IDictionary2<TToken>>
+    //                   |                  \_______________[*]IDictionary2<Recipient, object?>       /
+    //                   |                                           \_________/_________/___        / 
+    //                   |\                       _(recipients registrations)_/         /    \      /
+    //                   | \__________________   /    _____(channel registrations)_____/______\____/
+    //                   |                    \ /    /      __________________________/        \
+    //                   |                     /    /      /                                    \
+    //                   |      Dictionary2<Recipient, object?> mapping = Mapping________________\
+    //                   | __________________/    /                |         /                    \
+    //                   |/                      /                 |        /                      \
+    //    Dictionary2<Recipient, Dictionary2<TToken, object?>> mapping = Mapping<TToken>____________\
+    //                                         /                  /       /  /
+    //                   ___(Type2.TToken)____/                  /       /  /
+    //                  /________________(Type2.TMessage)_______/_______/__/
+    //                 /       ________________________________/
     //                /       /
     // Dictionary2<Type2, IMapping> typesMap;
     // --------------------------------------------------------------------------------------------------------
-    // Each combination of <TMessage, TToken> results in a concrete Mapping<TMessage, TToken> type, which holds the references
-    // from registered recipients to handlers. The handlers are stored in a <TToken, MessageHandler<object, TMessage>> dictionary,
-    // so that each recipient can have up to one registered handler for a given token, for each message type.
-    // Note that the registered handlers are only stored as object references, even if they were actually of type
-    // MessageHandler<TRecipient, TMessage>, to avoid unnecessary unsafe casts. Each handler is also generic with respect to the
-    // recipient type, in order to allow the messenger to track and invoke type-specific handlers without using reflection and
-    // without having to capture the input handler in a proxy delegate, causing one extra memory allocations and adding overhead.
+    // Each combination of <TMessage, TToken> results in a concrete Mapping type (if TToken is Unit) or Mapping<Token> type,
```

Comment:
Maybe look at splitting this up into multiple paragraphs to improve readability

Reply:
Done in bf0fab6fa3bf1e8c2336ce352ed276cc739cea07 👍